bots/bot.go

```diff
@@ -0,0 +1,36 @@
+package bot
+
+import "crypto/tls"
+
+type BotHandleFunc func(string, []byte)
+
+type Bot interface {
+	HandleRegistration(string, []byte) string
+	Run()
+	Stop()
+}
+
+func SendRequestToCONIKS(address string, msg []byte) ([]byte, error) {
+	conf := &tls.Config{
+		InsecureSkipVerify: true,
```

Comment:
> The reason I would like to use TLS connection here is to prevent (say) malicious sysadmins from intercepting the connection between the bots and the server, and thus to preserve the privacy of the protocol.

But if the sysadmin has access to both TLS certs, the connection can still be intercepted, no?